layouts/shortcodes/expand.html

```diff
@@ -1,17 +1,20 @@
-{{ $_hugo_config := `{ "version": 1 }` }}
-<div class="expand">
-  <div class="expand-label" style="cursor: pointer;" onclick="$h = $(this);$h.next('div').slideToggle(100,function () {$h.children('i').attr('class',function () {return $h.next('div').is(':visible') ? 'fas fa-chevron-down' : 'fas fa-chevron-right';});});">
-    <i style="font-size:x-small;" class="fas fa-chevron-right"></i>
-      <span>
-      {{$expandMessage := "Click to expand"}}
+<details closed class="expand">
+  <summary class="expand-label" style="cursor: pointer;">
+    <span>
       {{- if .IsNamedParams -}}
-        {{.Get "default" | default $expandMessage}}
-      {{- else -}}
-        {{.Get 0 | default $expandMessage}}
-      {{- end -}}
-      </span>
+      {{.Get "default" | default "Click to expand"}}
+    {{- else -}}
+      {{.Get 0 | default "Click to expand"}}
+    {{- end -}}
+    </span>
+  </summary>
+  <div class="expand-content">
+    {{- if (findRE "(gif|mp4)_src" .Inner) -}}
```

Comment:
and what does this do? It looks for gifs and mp4s and if so wraps it with safeHTML and otherwise uses markdownify? Is that needed for edgecases?